monitoring.md

```diff
@@ -226,3 +226,17 @@ the `amppackager` server binary. Every request to `/metrics` is served with the
 stats accumulated since the server's been up, up to the time of the request, but
 not including the request itself. 
 
+## Example alerts
+
+Below are a few examples of indicators of possibly abnormal behavior of
+`amppackager` and/or the underlying AMP document server. Feel free to adjust the
+numbers and check these manually, or setup automatic alerts in Prometheus:
+
+* Non-200 responses count going beyond 1% of all requests.
+* Latencies .9 percentile going beyond 60 seconds (of either server).
```

Comment:
change to `90 percentile` (alternatively `.9 quantile` maybe?)

Reply:
Done.